client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -396,6 +404,67 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
         };
     }
 
+    /**
+     * Function to get a list with all favorite motions.
+     *
+     * @returns An array containing all favorite motions.
+     */
+    public motionsAsFavorite(): ViewMotion[] {
+        return this.motionRepo.getSortedViewModelList().filter(motion => motion.star);
+    }
+
+    /**
+     * Function to get a list of contained motions in a category.
+     *
+     * @param category The category
+     *
+     * @returns An array with all the motions in the given category.
+     */
+    public motionsInCategory(category: Category): ViewMotion[] {
+        return this.motionRepo.getSortedViewModelList().filter(motion => motion.category_id === category.id);
+    }
+
+    /**
+     * Function to get the favourite view for the user.
+     */
+    private async initializeView(): Promise<void> {
+        this.selectedView = (await this.storage.get<string>('motionListView')) || 'tiles';
```

Comment:
Please move this line to the `ngOnInit`. One function for one line is not necessary..